content/intermediate/241_pixie/using_pixie/http_data.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Inspect the HTTP Request"
+date: 2021-5-01T09:00:00-00:00
+weight: 23
+draft: false
+---
+
+Pixie automatically traces the full request/response bodies of application w for the following [supported protocols](https://docs.pixielabs.ai/about-pixie/observability/). Let’s use Pixie to see the contents of this failed HTTP request.
+
+### Inspect the HTTP Request
+
+Select the `px/http_data` script from the script drop-down menu. Change the `start_time` to `-30m` or any window that will include when you [triggered the bug](/intermediate/241_pixie/prereqs/#trigger-the-microservices-application-bug) in the Sock Shop app.
+
+This script shows the traced HTTP1/2 requests made within your cluster, as long as the request either originates or is received inside of your cluster. There is quite a bit of HTTP traffic in this cluster, so let’s filter it to only show errors.
+
+Open the script editor using `ctrl+e` (Windows, Linux) or `cmd+e` (Mac). Or by clicking on the edit script button circled in red below.
+
+![script_editor](/images/pixie/script_editor.png)
+
+Delete line 29 and replace it with the following.
```

Comment:
I copy/pasted without regard to indentation, and saw an error. I think we should mention that the new lines should be spaced to match the existing spacing.